lib/Agrammon/Web/Routes.pm6

```diff
@@ -451,6 +451,14 @@ sub application-routes(Agrammon::Web::Service $ws) {
             content 'application/json', %cfg;
         }
 
+        post -> Agrammon::Web::SessionUser $user, 'logout' {
+            $user.logout();
+            content 'application/json', %(
+                :user($user.username),
+                :!sudoUser,
```

Comment:
Hm, why is this needed?

Reply:
> Hm, why is this needed?

Do you mean the `:!sudoUser`? If so, that's something that is not implemented yet. The GUI has a `sudo` option for role `admin` so that an admin user can become another user without having to know his password (for supporting "help, my dataset doesn't work"). Upon `logout` the admin user is then returned to his own login account (and thus the GUI needs to know that it has to display the admin's username again and not open the login window.

Not sure if this is a bad idea from a security point of few. But I thought we could discuss this once I have implemented it.

Reply:
Yes, about `sudoUser`. The question was why that could ever matter at logout time; now I understand why!